components/config/prefs.rs

```diff
@@ -299,6 +299,8 @@ mod gen {
                     test: bool,
                     #[serde(default)]
                     glwindow: bool,
+                    #[serde(default)]
+                    hand: bool,
```

Comment:
should the pref be `dom.webxr.hand` or `dom.webxr.hand.enabled`?

Reply:
done, `dom.webxr.hands.enabled` (the API is called webxr-hands-input, so I picked this option instead)